components/SpotifyInterface/SpotifyMakeRequest.c

```diff
@@ -257,22 +322,49 @@ void Spotify_GetCurrentPlaying(Token_t *Token)
  * @param[in] code is parameter that we give it before .
  * @return This function does not return a value.
  */
-void SendRequest_ExchangeTokenWithRefreshToken(char *Buf, size_t SizeBuf, char *RefreshToken_t)
+void SendRequest_ExchangeTokenWithRefreshToken(char *Buf, size_t SizeBuf, char *refreshToken)
```

Comment:
If you don't nedd Buf , and SizeBuf please remove them , and change "refreshToken" to "RefreshToken"

Reply:
good point, I removed Buf
for local variables, we use camelCase